### PR TITLE
new api.namespaces settings

### PR DESCRIPTION
**Pull request by jmazzitelli**
part of: https://github.com/kiali/kiali/issues/5516

server PR: https://github.com/kiali/kiali/pull/5521
operator PR: https://github.com/kiali/kiali-operator/pull/574
doc PR: https://github.com/kiali/kiali.io/pull/587

### CODE REVIEW

**Comment by jmazzitelli** [comment]
A quick test to make sure nothing broke - run the molecule test that exercises the label selector (now label_selector_include)

1. Install minikube and Istio
2. `make -e CLUSTER_TYPE=minikube -e MOLECULE_SCENARIO="accessible-namespaces-test" -e MOLECULE_USE_DEV_IMAGES="true" build build-ui cluster-push molecule-test`

That test should pass.

**Comment by jmazzitelli** [comment]
all tests pass on minikube and openshift (see https://github.com/kiali/kiali/pull/5521#issuecomment-1271713489) so this is good to go.

File: roles/default/kiali-deploy/tasks/main.yml [hunk 1]
```diff
@@ -91,6 +91,15 @@
   - kiali_vars.deployment.ingress_enabled is defined
   - kiali_vars.deployment.ingress is not defined or kiali_vars.deployment.ingress.enabled is not defined
 
+- name: api.namespaces.label_selector is deprecated but if api.namespaces.label_selector_include is not set then use the old setting
```
Comment (jshaughn) [inline roles/default/kiali-deploy/tasks/main.yml:94]: Good.

**Review by jshaughn — APPROVED** [review]
LOK!

**Pull request merged**